psalm.xml

```diff
@@ -28,6 +28,11 @@
 				<referencedClass name="Doctrine\DBAL\Types\Types" />
 				<referencedClass name="OC" />
 				<referencedClass name="OCA\Guests\UserBackend" />
+				<referencedClass name="Symfony\Component\Console\Command\Command" />
```

Comment:
Not a fan of this thirdparty access. Symfony version is not the same across all supported versions.
Also by ignoring it in psalm, CI will not detect if there is an issue. But yeah we don't have a better option atm.